sway-fmt/src/fmt.rs

```diff
@@ -641,4 +641,23 @@ fn one_liner() -> bool {
         let (_, formatted_code) = result.unwrap();
         assert_eq!(correct_sway_code, formatted_code);
     }
+
+    #[test]
+    // This test is here to check that the `use` statements are properly formatted
+    // Currently, there is a hard-to-diagnose bug which adds a leading `--` to the first
```

Comment:
It's actually not hard-to-diagnose, 1st phase of the formatter adds some _metadata_ in the form of patterns like "---", and this bug is introduced since single use-statement is executed 2 or more times after Sway got updated with the ability to import multiple dependencies within a single statement.

Reply:
Yeah I forgot to remove those, this was during debugging phase